steampy/login.py

```diff
@@ -33,6 +34,15 @@ def _send_login_request(self) -> requests.Response:
         request_data = self._prepare_login_request_data(encrypted_password, rsa_timestamp)
         return self.session.post(self.STORE_URL + '/login/dologin', data=request_data)
 
+    # set the "sessionid" cookie for the domain "steamcommunity.com"
```

Comment:
Comment shouldnt be here, method name documents itself. Its nice clean function.